modules/pubmaticBidAdapter.js

```diff
@@ -326,6 +362,11 @@ export const spec = {
           utils.logWarn(BIDDER_CODE + ': Skipping the non-standard adslot: ', bid.params.adSlot, bid);
           return;
         }
+      } else if (bid.params.hasOwnProperty('native')) {
+        // TODO : Check for valid ad slot in native
+        if (!bid.params.native.assets || bid.params.native.assets.length < 1) {
+          return;
```

Comment:
can you add a log message here indicating that issue.

Reply:
Done